data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
While I still stand by the statement that this is a horrible idea, in that specific case there's an excuse of bleach drinking being already in the game.
A horrible idea with an excuse is a step above horrible and inexcusable idea.